libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/milestone/VoiceInstructionMilestone.java

```diff
@@ -27,7 +27,9 @@
   public boolean isOccurring(RouteProgress previousRouteProgress, RouteProgress routeProgress) {
     if (isNewRoute(routeProgress)) {
       clearInstructionList();
-      voiceInstructionLoader.cacheInstructions(routeProgress, true);
+      if (voiceInstructionLoader != null) {
```

Comment:
Can we extract this into a `private` method?  Something like `cacheInstructionsFromProgress` to remove the duplicate code here?